R/package.R

```diff
@@ -77,9 +77,8 @@ NULL
 #'   spaces to delimit arguments like you would in the shell). For example,
 #'   `build_args = c("--force", "--keep-empty-dirs")` is a correct usage and
 #'   `build_args = "--force --keep-empty-dirs"` is incorrect.
-#' @param check_dir Path to a directory where the check is performed.
-#'   If this is not `NULL`, then a temporary directory is used, that
-#'   is cleaned up when the returned object is garbage collected.
+#' @param check_dir Path to a directory where the check is performed. If this is
+#'   `NULL`, then a temporary directory is used, that is cleaned up afterwards.
```

Comment:
I'd suggest this:

```suggestion
#' @param check_dir Path to a directory where the check will be performed. 
#'   Will be created if needed. 
#'
#'   Alternatively, supply `NULL` to perform the check in a temporary directory
#'   that will be automatically cleaned up when the object returned by `rcmdcheck()`
#'   is deleted. 
```

@gaborcsardi the biggest problem with the current text is that it says "if *not* `NULL`", but I think the whole thing is easier to understand if broken up into two clauses.

Reply:
Thank you for the suggestion. I committed it.